photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java

```diff
@@ -341,6 +341,11 @@ public Optional<EstimatedRobotPose> update(
         return update(cameraResult, cameraMatrix, distCoeffs, this.primaryStrategy);
     }
 
+    private Optional<EstimatedRobotPose> update(
```

Comment:
Do we need this overload? if so, document

Reply:
It's not strictly necessary, but it's a bit nice not to have to plop `Optional.empty()` in all of the fallback calls to `update()`. Doc comment has been added, and if you like I could remove it instead.